lightningd/plugin.h

```diff
@@ -196,6 +194,18 @@ bool plugin_paths_match(const char *cmd, const char *name);
  */
 bool plugin_remove(struct plugins *plugins, const char *name);
 
+/**
+ * Kick of initialization of a plugin.
+ */
+bool plugin_send_getmanifest(struct plugin *p);
+
+/**
+ * Kick of initialization of all plugins which need it/
+ *
+ * Return true if any were started.
+ */
+bool plugins_send_getmanifest(struct plugins *plugins);
```

Comment:
Why does this need to be public?

Reply:
Called for dynamic plugins too (startdir).